homeassistant/components/yale_smart_alarm/config_flow.py

```diff
@@ -45,28 +44,29 @@ class YaleConfigFlow(ConfigFlow, domain=DOMAIN):
 
     VERSION = 1
 
-    entry: ConfigEntry
+    entry: ConfigEntry | None
 
     @staticmethod
     @callback
     def async_get_options_flow(config_entry: ConfigEntry) -> YaleOptionsFlowHandler:
         """Get the options flow for this handler."""
         return YaleOptionsFlowHandler(config_entry)
 
-    async def async_step_import(self, config: dict):
+    async def async_step_import(self, config: dict[str, Any]) -> FlowResult:
```

Comment:
Type hint for `config` can be `ConfigType` I think.

Reply:
Sure but don't seem to be common practice to use ConfigType as hint in the config flow.

Reply:
You are right.  The same [query came up here and was answered](https://github.com/home-assistant/core/pull/64193#discussion_r785454187).

What you have agrees with `script/scaffold/templates/config_flow/integration/__init__.py` which I assume is the reference.